### ISSUE TITLE
How many epoch did you train?

### ISSUE THREAD

**Issue by juierror** [Fengdalu/LipNet-PyTorch#1]
Now, I try to reproduce LipNet paper for my undergraduate senior project.

I have some question about How many epoch did you train?

PS1: I implement from paper and get output like the same thing. Did you ever found this problem?

**Best regard**

**Comment by Fengdalu** [comment]
nearly 200 - 300 epochs. It takes about 24hr training on my titan xp 12g gpu device at batch size 200 with adam optimizer lr=1e-4. It is normal when model outputs same characters at the begenning of the training. Hopes my experience may help you.

**Comment by kramer425** [comment]
Hey @Fengdalu, thanks for the implementation. Did you use a lr of 1e-4 the whole time or did you decrease it after a certain point? I'm currently at epoch 195 with a val loss of 0.45 and all the predictions are still the same.
